src/index.ts

```diff
@@ -41,7 +41,11 @@ export function startActivity(state: LiveActivityState, config?: LiveActivityCon
   if (Platform.OS !== "ios") {
     throw new Error("startActivity is only available on iOS");
   }
-  return ExpoLiveActivityModule.startActivity(state, config);
+  try {
+    return ExpoLiveActivityModule.startActivity(state, config);
+  } catch (error) {
+    throw new Error(`startActivity failed with an error: ${error}`);
```

Comment:
Instead of throwing errors, let's just do `console.error`
```suggestion
    console.error(`startActivity failed with an error: ${error}`);
```